GameDefinition/PLAYING.md

```diff
@@ -527,6 +539,90 @@ influence the speed of HP attrition and if it outweighs the benefits
 of exploration.
 
 
+Managing inventory and crafting
+-------------------------------
+
+All items belonging to your team are stored either in the character's personal 
+equipment outfit or in the shared inventory stash that is accessible by every 
+team member. 
+Items that are in the personal equipment outfit can be accessed by or 
+are beneficial for this particular character only. Stashed items can be 
+accessed by the team members (unless the character is not calm enough 
+or injured). But no one has a direct advantage from having the items 
+in the shared inventory stash.
+The stashed items are exactly stashed, they are not simply laying on
+the ground (although the stash has a particular location on the map). You
+cannot operate the items from the stash, as if these items are just on
+the ground directly. To collect an item from the stash or drop it on the ground,
```

Comment:
I find the sentence "You cannot operate the items from the stash, as if these items are just on the ground directly." spurious and confusing, though I know what you mean. Perhaps remove it and instead extend the one: "you should use commands that refer to the party stash, not items on the ground."

Reply:
Changed. Hopefully, it will look better